test/wallet-http-test.js

```diff
@@ -79,6 +79,7 @@ describe('Wallet HTTP', function() {
   });
 
   beforeEach(async () => {
+    mineBlocks = common.constructBlockMiner(node, nclient);
```

Comment:
We don't need to do this before every `it` block do we? Its always the same node and the same client

Reply:
Nope!